lib/eslint.js

```diff
@@ -11,10 +11,10 @@ const EngineConfig = require("./engine_config")
 const checks = require("./checks")
 const computeFingerprint = require("./compute_fingerprint")
 
-const CLIEngine = patch.eslint.CLIEngine
-const options = { extensions: [".js"], ignore: true, reset: false, useEslintrc: true, baseConfig: {} }
+const ESLint = patch.eslint.ESLint
+const options = { extensions: [".js"], ignore: true, useEslintrc: true, baseConfig: {} }
```

Comment:
why is `reset: false` no longer present in `options` ?

Reply:
The `reset` option is [no longer available](https://eslint.org/docs/developer-guide/nodejs-api)